### ISSUE TITLE
Running --vary_name breaks paramVary folder

### ISSUE THREAD

**Issue by giovannivizcarra** [chill90/BoloCalc#5]
1. ![Screen Shot 2020-04-08 at 7 07 45 PM](https://user-images.githubusercontent.com/62725355/78850521-49ff9f80-79cc-11ea-828e-053e6af17ed1.png)

After running the following code:
"python calcbolos.py --vary --vary_name insertname experiments/exampleexperiment/v0/ "
the paramVary folder would no longer update for any of the projects unless I used --vary --vary_name code again again as seen in image 1, where the last code I ran in the image was able to update the paramvary folder, but prior to that I was having issues with updating the folder since I was no longer using the --vary_name argument.

 In other words when this keyword argument is not specified, then there is no vary parameter output sent to the directory called [date] [hour] [min] [sec].

**Comment by giovannivizcarra** [comment]
Note: The file does update when to the general [date] [hour] [min] [sec] but now I have to run 
"python calcbolos.py --vary  experiments/exampleexperiment/v0/ " for it to update whereas before I could run "python calcbolos.py experiments/exampleexperiment/v0/ " and it would work

**Comment by giovannivizcarra** [comment]
nvm realized my mistake
